modules/rosapps/applications/screensavers/butterflies/butterflies.c

```diff
@@ -285,8 +286,9 @@ LRESULT WINAPI ScreenSaverProc(HWND hWnd, UINT message,
 		ReleaseDC(hWnd, hDC);
 		break;
 	case WM_DESTROY:
-		wglMakeCurrent(NULL, NULL);
+		wglMakeCurrent(hdcOpenGL, NULL);
 		wglDeleteContext(hRC);
+		ReleaseDC(hWnd, hdcOpenGL);
```

Comment:
It could be done without adding global `hdcOpenGL` variable, by using `wglGetCurrentDC()`.

So you can do this:
```c
		hDC = wglGetCurrentDC(); // hDC is a local variable in this function
		wglMakeCurrent(NULL, NULL); // this is correct
		wglDeleteContext(hRC);
		ReleaseDC(hWnd, hDC);
```

Reply:
We don't do that in our other screensavers, does it add an advantage?